var/spack/repos/builtin/packages/superchic/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Superchic(MakefilePackage):
+    """SuperChic is a Fortran based Monte Carlo event generator for exclusive and
+       photon-initiated production in proton and heavy ion collisions. A range of
+       Standard Model final states are implemented, in most cases with spin
+       correlations where relevant, and a fully differential treatment of the soft
+       survival factor is given. Arbitrary user-defined histograms and cuts may be
+       made, as well as unweighted events in the HEPEVT, HEPMC and LHE formats"""
```

Comment:
Leaving a short first paragraph lets Spack use that wherever a short description is needed. The entire docstring will be used instead in context where a long description fits better.
```suggestion
    """SuperChic is a Fortran based Monte Carlo event generator for exclusive and
    photon-initiated production in proton and heavy ion collisions. 
    
    A range of Standard Model final states are implemented, in most cases with 
    spin correlations where relevant, and a fully differential treatment of the soft
    survival factor is given. Arbitrary user-defined histograms and cuts may be
    made, as well as unweighted events in the HEPEVT, HEPMC and LHE formats.
    """
```